src/Backend/Modules/Faq/Domain/Category/Category.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+namespace Backend\Modules\Faq\Domain\Category;
+
+use Common\Doctrine\Entity\Meta;
+use Common\Locale;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+use Exception;
+
+/**
+ * @ORM\Entity(repositoryClass="Backend\Modules\Faq\Domain\Category\CategoryRepository")
+ * @ORM\Table(name="faq_categories")
```

Comment:
This is more a general question for the @forkcms/core-contributors. In my opinion this is the moment to change these tables into a more Doctrine-style name. eg: FaqCategories.

I know we discussed to not change the table names, but as we have changed the fields I don not see the point of keeping the table name.

Reply:
agree

Reply:
I'm not a big fan of using "plurals" as table names.
I prefer singular - entity based - names (Module + Domain/Entity name), because they are easier to understand and expand in bigger modules.

Example:
```
FaqCategory
FaqComment
FaqQuestion
```

Especially when you have big modules, plurals can cause confusion in how to name the tables... Because you then have to answer the following question:
```
FaqCategories
FaqCategoriesChild OR FaqCategoryChild?
Pages
PagesBlock OR PageBlock ?
... etc...
```